File: cloud/dynamic-site/gt-cs-1331-py/app/gtcs1331/solutions/solution106.py
'''Solution 1.6 implementation'''
from functools import reduce
from gtcs1331.solution_api.solution import Solution

class Solution106(Solution):
    '''Solution 1.6'''
    id = "1.6"
    title = "Summation series"

    def __answer(self):
        result = reduce(lambda a,b:a+b, range(1,10))
        return [f"1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 = {result}"]

    answer = property(__answer)

File: cloud/dynamic-site/gt-cs-1331-csharp/solutions/Solution106.cs
using _4MeOnWeb.SolutionAPI;

namespace _4MeOnWeb.Solutions {
    public class Solution106 : ImmutableSolution {
        public Solution106() :
            base("1.6", "Summation series") {
        }

        public override IEnumerable<string> Answers {
            get {
                int result = Enumerable.Range(1, 9).Sum();
                return new string[] { $"1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9 = {result}" };
            }
        }
    }
}